BodoSQL/calcite_sql/bodosql-calcite-application/src/main/java/com/bodosql/calcite/table/BodoSQLColumn.java

```diff
@@ -98,60 +96,13 @@ public final String getTypeIdName() {
       return this.type_id_name;
     }
 
-    // Note: This is a Python facing API.
     public static BodoSQLColumnDataType fromTypeId(int typeId) {
       for (BodoSQLColumnDataType verbosity : BodoSQLColumnDataType.values()) {
         if (verbosity.getTypeId() == typeId) return verbosity;
       }
       throw new RuntimeException(String.format(Locale.ROOT, "Unknown type id: %d", typeId));
     }
 
-    // Note: This isn't currently used but would be used if we get type information via
-    // a generic JDBC connection, so its here for completeness.
-    public static BodoSQLColumnDataType fromJavaSqlType(final JDBCType typeId) {
```

Comment:
This code presents risk since it's untested and unused. I think its best to remove it and just check the git history if we decide to add it in the future.